docs/USAGE.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: documentation
+title: Android openHAB App
+---
+
+{% include base.html %}
+
+# Android openHAB App
+
+openHAB Android application is a native client for openHAB. It uses REST API of openHAB to render your sitemaps.
+
+<a href="https://play.google.com/store/apps/details?id=org.openhab.habdroid"><img src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" height="80"></a>
+
+## Features
+* Control your openHAB server and [openHAB Cloud instance](https://github.com/openhab/openhab-cloud)
+* Receive notifications from openHAB Cloud
+* Change items via NFC tags
+* Send voice commands to openHAB
+* Discover devices and add them as items
+* Supports wall mounted tablets
+
+<img src="images/main_menu.png" width="200px"> <img src="images/widget_overview.png" width="200px">
+
+## Getting Started
+
+When first installed the app is in "Demo Mode".
+To connect it to your own openHAB server, first navigate to Settings and uncheck the "Demo Mode" option.
+Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to autodetect to your openHAB server if it is on the same network.
+
+You also have the option to manually set the server URL in the settings.
+Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
```

Comment:
Maybe add two highlights here:

> **Local Connection:**  Enter the base URL of ...
>
> **Remote Connection:** The option to set a "Remote URL" ...

Reply:
Wdyt?

Reply:
I think it is clear enough what to enter.

Reply:
It's not about clear or not, those paragraph names will add more structure to the article.

Reply:
Added the structure.